README.md

```diff
@@ -16,17 +15,19 @@ of `TypeMap`.
 extern crate typemap;
 use typemap::{TypeMap, Key};
 
-struct KeyType;
+// Some arbitrary type that we will use as a Key in Typemap
+struct MyKeyType;
 
 #[derive(Debug, PartialEq)]
-struct Value(i32);
+struct MyValueType(i32);
 
-impl Key for KeyType { type Value = Value; }
+// The value for the type Key MyKeyType must be an instance of MyValueType
```

Comment:
Hmm, this doesn't sound very clear to me; how about "Define the key-value relationship between MyKeyType and MyValueType."?